cirq-google/cirq_google/common_serializers.py

```diff
@@ -652,6 +652,66 @@ def _can_serialize_limited_iswap(exponent: float):
 #
 #############################################
 
+#
+# Coupler Pulse serializer and deserializer
+#
+
+COUPLER_PULSE_SERIALIZER = op_serializer.GateOpSerializer(
+    gate_type=CouplerPulse,
+    serialized_gate_id='pulse',
+    args=[
+        op_serializer.SerializingArg(
+            serialized_name='coupling_mhz', serialized_type=float, op_getter='coupling_mhz'
+        ),
+        op_serializer.SerializingArg(
+            serialized_name='hold_time',
```

Comment:
include units here: `hold_time_ns`? Same thing for other duration args below.

Reply:
Done.